meetuppizza/tests.py

```diff
@@ -36,9 +36,13 @@ def test_user_is_created(self):
     self.assertFalse(user == None)
 
   def test_super_user_is_created(self):
-    self.client.post('/sign_up', self.params)
     user = User.objects.get(username='adminotaur')
-    self.assertFalse(user == None)
+    self.assertTrue(user.is_staff)
```

Comment:
:+1: for tighter assertions here: these actually test that the user is a superuser, not just that some user named `adminotaur` exists. Always double check that the name of your test and the assertions inside actually match up.